systemvm/noVNC/include/keyboard.js

```diff
@@ -0,0 +1,543 @@
+var kbdUtil = (function() {
+    "use strict";
```

Comment:
Which keyboard layout comes default, and is there a way to switch to other keyboard layouts?